docs/json/sonarr/cf/ovid.json

```diff
@@ -0,0 +1,35 @@
+{
+  "trash_id": "fb1a91cdc0f26f7ca0696e0e95274645",
```

Comment:
Sonarr streaming services have a default score of 50 which needs adding here.

Reply:
Added